clipmenu

```diff
@@ -1,7 +1,17 @@
 #!/bin/bash
 
-: "${CM_LAUNCHER=dmenu}"
 : "${CM_DIR="${XDG_RUNTIME_DIR-"${TMPDIR-/tmp}"}"}"
+# Pick dmenu or rofi, depending on what's available on the system
+if [[ -z "$CM_LAUNCHER" ]]; then
```

Comment:
if ! [[ -v CM_LAUNCHER ]]; then

Reply:
Why? This is further from POSIX syntax I'm used to. Is it faster?...

Reply:
We already use lots of non-POSIX features, so trying to keep close compatibility isn't a goal. `-v` has clearer intent than `-z`. This isn't a massive deal but I'd rather stick to options which semantically represent the intent more clearly.